### DISCUSSION TITLE
Multiple javascript errors

### DISCUSSION THREAD

**Discussion by solidevolution** [tinacms/tinacms#2711] [Help]
There are multiple javascript errors and warnings in a new project created by cli.

<img width="1511" alt="Bildschirmfoto 2022-03-20 um 19 39 33" src="https://user-images.githubusercontent.com/6850773/159177452-e5a1c411-2dcf-4b59-9371-4b1dcdb2cb94.png">
<img width="1509" alt="Bildschirmfoto 2022-03-20 um 19 39 18" src="https://user-images.githubusercontent.com/6850773/159177455-9e92764b-fec3-4c61-ba81-e1ccfc037760.png">
<img width="1509" alt="Bildschirmfoto 2022-03-20 um 19 39 09" src="https://user-images.githubusercontent.com/6850773/159177456-3aa72f43-219b-4c80-9b05-adec6c853d7e.png">

**Comment by perkinsjr** [comment]
Hi this should be resolved with a PR, thank you for providing the feedback. Note these errors won't affect the editing nor usage of the app.
